<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[testLogin01 Failed. Link to Screenshot: ]]>
    </line>
    <line>
      <![CDATA[<a href="/Users/rameshtejavath/Documents/Selenium/Batch1115/Catalog/screenshots\eCommerge\Cycle1\screenshots\12-05-2015_11.20.45_AccountLoginTests_testLogin01.jpg">12-05-2015_11.20.45_AccountLoginTests_testLogin01.jpg</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="10749" started-at="2015-12-05T19:20:35Z" finished-at="2015-12-05T19:20:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10749" started-at="2015-12-05T19:20:35Z" finished-at="2015-12-05T19:20:46Z">
      <class name="com.ecalix.tests.AccountLoginTests">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.ecalix.tests.AccountLoginTests@abeafe7]" name="setup" is-config="true" duration-ms="3398" started-at="2015-12-05T11:20:35Z" finished-at="2015-12-05T11:20:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FireFox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="navigate(java.lang.String)[pri:0, instance:com.ecalix.tests.AccountLoginTests@abeafe7]" name="navigate" is-config="true" duration-ms="651" started-at="2015-12-05T11:20:38Z" finished-at="2015-12-05T11:20:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FireFox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigate -->
        <test-method status="FAIL" signature="testLogin01()[pri:0, instance:com.ecalix.tests.AccountLoginTests@abeafe7]" name="testLogin01" duration-ms="5128" started-at="2015-12-05T11:20:39Z" finished-at="2015-12-05T11:20:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Welcome to osCommerce Demo] but found [Welcome to iBusiness]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Welcome to osCommerce Demo] but found [Welcome to iBusiness]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.ecalix.pageobjects.WelcomePage.<init>(WelcomePage.java:50)
	at com.ecalix.tests.AccountLoginTests.testLogin01(AccountLoginTests.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin01 -->
        <test-method status="PASS" signature="after(org.testng.ITestResult)[pri:0, instance:com.ecalix.tests.AccountLoginTests@abeafe7]" name="after" is-config="true" duration-ms="1550" started-at="2015-12-05T11:20:44Z" finished-at="2015-12-05T11:20:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLogin01 status=FAILURE method=AccountLoginTests.testLogin01()[pri:0, instance:com.ecalix.tests.AccountLoginTests@abeafe7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[testLogin01 Failed. Link to Screenshot: ]]>
            </line>
            <line>
              <![CDATA[<a href="/Users/rameshtejavath/Documents/Selenium/Batch1115/Catalog/screenshots\eCommerge\Cycle1\screenshots\12-05-2015_11.20.45_AccountLoginTests_testLogin01.jpg">12-05-2015_11.20.45_AccountLoginTests_testLogin01.jpg</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- com.ecalix.tests.AccountLoginTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
